kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/config/BrokerAddressPatternUtils.java

```diff
@@ -18,7 +18,7 @@
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 
-final class BrokerAddressPatternUtils {
+public final class BrokerAddressPatternUtils {
```

Comment:
I opened up the methods during the refactor as I moved classes around, it can be locked back down I think